src/routes/dad-joke.js

```diff
@@ -0,0 +1,44 @@
+import { Router } from 'express';
+
+const dadJokeRouter = Router();
+
+/**
+ * @swagger
+ * /dad-joke:
+ *   get:
+ *     summary: Get a random dad joke.
+ *     description: Returns a random dad joke.
+ *     tags: [Dad Joke]
+ *     responses:
+ *       200:
+ *         description: A random dad joke.
+ *         content:
+ *           text/plain:
+ *             schema:
+ *               type: string
+ *       500:
+ *         description: Internal Server Error.
+ *         content:
+ *           text/plain:
+ *             schema:
+ *               type: string
+ *               example: Joke load failed
+ */
+
+dadJokeRouter.get('/', async (req, res) => {
+  try {
+    const response = await fetch('https://icanhazdadjoke.com/', {
```

Comment:
In the Node.js environment, the fetch function is not globally available out-of-the-box, as it is in the browser environment. To use fetch in Node.js, it is necessary to install an external library, such as node-fetch.